src/electron-main.js

```diff
@@ -732,6 +733,17 @@ ipcMain.on('seshat', async function(ev, payload) {
             }
             break;
 
+        case 'resetEventStore':
+            err = await fs.promises.rmdir(eventStorePath, { recursive: true });
+            if (!err) {
+                console.log("Seshat Event Store successfully deleted");
+                app.relaunch();
+                app.exit();
```

Comment:
hmm, why a relaunch and an exit? Shouldn't the relaunch be enough?

Reply:
You would assume that calling `app.relaunch` would be enough... However reading electron's API docs for [`app.relaunch([options])`](https://www.electronjs.org/docs/api/app#apprelaunchoptions)

> Note that this method does not quit the app when executed, you have to call app.quit or app.exit after calling app.relaunch to make the app restart.

🤷‍♂️